sdks/python/apache_beam/pipeline_test.py

```diff
@@ -185,65 +180,6 @@ def apply(self, pcoll):
         ['a-x', 'b-x', 'c-x'],
         sorted(['a', 'b', 'c'] | 'AddSuffix' >> AddSuffix('-x')))
 
-  def test_cached_pvalues_are_refcounted(self):
```

Comment:
I have not added an equivalent test. Counts does not work, because bundling may cause more object references. There is also no cache object to be tested. An option might be to check memory usage before and after p.run and make sure that result does not have any references to the intermediate results.

Any suggestions?